server/app/services/openApi/OpenApiVersion.java

```diff
@@ -0,0 +1,36 @@
+package services.openApi;
+
+import java.util.Optional;
+
+public enum OpenApiVersion {
+  SWAGGER_V2(VersionLabels.swagger_v2),
+  OPENAPI_V3(VersionLabels.openapi_v3);
+
+  // Nest class is here as a hack/workaround because you can't put the strings directly in the enum
```

Comment:
Yeah, I was trying to avoid duplicating magic strings and probably went a little overboard. I can switch it:

```java
public enum OpenApiVersion {
  SWAGGER_V2("swagger-v2"),
  OPENAPI_V3("openapi-v3");

  public static OpenApiVersion fromStringOrDefault(Optional<String> openApiVersion) {
    switch (openApiVersion.orElse("swagger-v2")) {
      case "openapi-v3":
        return OPENAPI_V3;
      case "swagger-v2":
      default:
        return SWAGGER_V2;
    }
  }

*trim*
```

Reply:
Done